<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutsDialog</class>
 <widget class="QDialog" name="ShortcutsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keyboard Shortcuts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Filter:</string>
       </property>
       <property name="buddy">
        <cstring>searchEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="E5ClearableLineEdit" name="searchEdit">
       <property name="toolTip">
        <string>Enter the regular expression that should be contained in the shortcut action</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Filter on</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="actionButton">
       <property name="toolTip">
        <string>Select to filter based on the actions</string>
       </property>
       <property name="text">
        <string>&amp;Action</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="shortcutButton">
       <property name="toolTip">
        <string>Select to filter based on shortcut or alternative shortcut</string>
       </property>
       <property name="text">
        <string>&amp;Shortcut or Alternative</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="shortcutsList">
     <property name="toolTip">
      <string>This list shows all keyboard shortcuts.</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;Keyboard Shortcuts List&lt;/b&gt;
&lt;p&gt;This list shows all keyboard shortcuts defined in the application. Double click an entry in order to change the respective shortcut. Alternatively, the shortcut might be changed by editing the key sequence in the respective column.&lt;/p&gt;</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Action</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Shortcut</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Alternativ</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>E5ClearableLineEdit</class>
   <extends>QLineEdit</extends>
   <header>E5Gui/E5LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchEdit</tabstop>
  <tabstop>actionButton</tabstop>
  <tabstop>shortcutButton</tabstop>
  <tabstop>shortcutsList</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShortcutsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
